src/pages/Storefront/Cart.js

```diff
@@ -106,9 +148,7 @@ const OrderSummary = ({ checkoutData }) => {
 // Cart page that shows the user the items in their cart, allows them to change quantity, and proceed to checkout
 const Cart = () => {
   const {
-    products: { /* loading: productsLoading, */ data: productsData },
-    checkout: { /* loading: checkoutLoading, */ data: checkoutData },
-    cartCount,
+    checkout: { loading, data },
   } = useShopify();
   // const team = useSelector(teamSelectors.selectTeam, shallowEqual);
```

Comment:
not sure if it is out of the scope of this PR, but I'm wondering how to best handle the team inside the checkout state (from #56)